frontend/app/work_packages/controllers/work-package-details-controller.js

```diff
@@ -151,9 +154,6 @@ module.exports = function($scope,
         relationTypeIterator(key);
       }
     }
-
-    // Author
-    $scope.author = workPackage.embedded.author;
```

Comment:
This is the same line as https://github.com/opf/openproject/pull/2701/files#diff-35a32f47df85900d4949ec429d7976dbL119